spring-2013/hw1/src/CreditCardValidator.hs

```diff
@@ -0,0 +1,22 @@
+module CreditCardValidator
+    ( toDigits
+    , doubleEveryOther
+    , sumDigits
+    , validate
+    ) where
+
+toDigits :: Integer -> [Integer]
+toDigits n
+    | n <= 0 = []
+    | otherwise = map readChar $ show n
+    where readChar = read . (:[])
```

Comment:
Interesting, I always put helper functions on their own lines:

```
where
    readChar = read . (:[])
```

I think it makes it easier to see that there's a helper.